docs/learn/governance/governance.md

```diff
@@ -0,0 +1,20 @@
+---
+id: governance
+title: Governance
+---
+
+
+Archethic Blockchain is using code proposals to have a way to upgrade current blockchain
+code in a automated and tested way. The idea behind the governance on Archethic blockchain is to be able to audit, test and apply code proposals in a automatic, indepedent and secure way across the deployed network.
+
+the process begins when a contributor creates a code proposal, this code proposal will be checked to ensure that it follows the proper form (contains the correct fields for example).
+
+After the code proposal is verified, we start going over the details of the proposal and verifying that it contains correct version, valid appup and valid changes for example.
+
+When the code proposal is considered valid, and after gathering enough approvals from members of the technical council (more than the threshhold) we start the CI part.
```

Comment:
Probably we should detail about CI/CD, or at least do no use acronyms, and try to explain in simple words what the purpose without going too much in the technical terms and details.